src/Recommender/Service/RecommendedUsers.jsx

```diff
@@ -0,0 +1,195 @@
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+  }
+
+  @media (max-width: 800px) {
+    grid-template-columns: minmax(0, 1fr);
+  }
+`;
+
+const RecommendedUsers = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  padding-bottom: ${props.sidebar ? "12px" : "50px"};
+`;
+
+State.init({
+  currentPage: 1,
+  isLoading: true,
+  error: null,
+  totalPages: 1,
+  displayedUsers: [],
+});
+
+const updateState = (data, totalPageNum) => {
+  State.update({
+    isLoading: false,
+    displayedUsers: [...state.displayedUsers, ...data],
+    totalPages: totalPageNum,
+  });
+};
+
+State.update({
+  displayedUsers: props.returnElements
+    ? state.displayedUsers.slice(0, props.returnElements)
+    : state.displayedUsers,
+});
+
+const passedContext = props.fromContext;
+const fromContext = { ...passedContext, scope: props.scope };
+
+const STORE = "storage.googleapis.com";
+const BUCKET = "databricks-near-query-runner";
+const BASE_URL = `https://${STORE}/${BUCKET}/output/recommendations`;
+
+const accountId = props.accountId;
+const profile = props.profile || Social.get(`${accountId}/profile/**`, "final");
+const tags = Object.keys(profile.tags || {});
+const profileUrl = `#/near/widget/ProfilePage?accountId=${accountId}`;
+
+const getRecommendedUsers = (page) => {
+  try {
+    const url = `${props.dataset}_${page}.json`;
+    if (state.currentPage == 1) {
+      const res = fetch(url);
+      if (res.ok) {
+        const parsedResults = JSON.parse(res.body);
+        const totalPageNum = parsedResults.total_pages || 10;
+        updateState(parsedResults.data, totalPageNum);
+      }
+    } else {
+      asyncFetch(url).then((res) => {
+        if (res.ok) {
+          const parsedResults = JSON.parse(res.body);
+          const totalPageNum = parsedResults.total_pages || 10;
+          updateState(parsedResults.data, totalPageNum);
+        }
+      });
+    }
+  } catch (error) {
+    console.log(error.message);
+  }
+};
+
+const loadMore = () => {
+  const nextPage = state.currentPage + 1;
+  if (nextPage <= state.totalPages) {
+    State.update({ currentPage: nextPage });
+    getRecommendedUsers(nextPage);
+  }
+};
+
+if (state.isLoading) {
+  getRecommendedUsers(state.currentPage);
+}
+
+const handleFollowed = (accountId) => {
+  const updatedUsers = state.displayedUsers.filter(
+    (user) => (user.recommended_profile || user.similar_profile) !== accountId
+  );
+  State.update({
+    displayedUsers: updatedUsers,
+  });
+};
+
+return (
+  <RecommendedUsers>
+    {state.isLoading && <p>Loading...</p>}
+    {state.error && <p>Error: {state.error}</p>}
+    <Profiles>
+      {!state.isLoading && state.displayedUsers.length < 4 ? (
+        <>
+          {!state.isLoading && (
+            <div>
+              Follow More Users to Unlock More Personalized Recommendations, See
+              Who’s
+              <a href="https://near.org/${REPL_ACCOUNT}/widget/PeoplePage?tab=trending">
```

Comment:
I think you can use `REPL_NEAR_URL` here:

```suggestion
              <a href="https://${REPL_NEAR_URL}/${REPL_ACCOUNT}/widget/PeoplePage?tab=trending">
```